.github/workflows/release.yml

```diff
@@ -121,4 +120,28 @@ jobs:
           export NEW_PYTHON_VERSION="999-dev0"
           .github/scripts/change_versions.sh
           git commit -am "build: move back to version $NEW_VERSION"
-          git push origin $RELEASE_BRANCH_NAME
\ No newline at end of file
+          git push origin $RELEASE_BRANCH_NAME
+
+      - name: Update release branch
+        working-directory: ./timefold-quickstarts
+        shell: bash
+        run: |
+          java_version=${{ github.event.inputs.javaVersion }}
+          version="${java_version%.*}.x"
+          echo $version
+          exists="$(git branch -a | grep -w $version || true)"
+          echo "branch $exists"
+          if [ -n "$exists" ]; then
```

Comment:
Why are we doing this if none of workflows in the other repos had this?
Can't we delete the branch if it exists, like we do everywhere else? For consistency.